corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -212,7 +217,9 @@ hqDefine("app_manager/js/details/case_claim", function () {
                 search_additional_relevant: self.searchAdditionalRelevant(),
                 search_button_display_condition: self.searchButtonDisplayCondition(),
                 search_command_label: self.searchCommandLabel(),
+                command_label_multimedia_image: $("#case-search-command-label-media input[type=hidden]").val() || null,
```

Comment:
This seems a bit odd, to be in a knockout model using jQuery to reach into another knockout model...but I don't have any immediate better suggestions. Curious if you looked at how pages like form/module settings do this?

Reply:
...nevermind, I looked back at the ticket and remembered that other parts of app manager serialize the entire form, which this code doesn't do.